.github/workflows/documentation.yml

```
@@ -17,4 +17,4 @@ on:
 jobs:
   documentation:
     name: "Documentation"
-    uses: "doctrine/.github/.github/workflows/documentation.yml@8.0.0"
+    uses: "greg0ire/doctrine-dot-github/.github/workflows/documentation.yml@remove-require"
```

You can now use ~11.0.0~ 12.0.0 of the main repo :)


```suggestion
    uses: "doctrine/.github/.github/workflows/documentation.yml@12.0.0"
```